tests/test_fyle_integrations_imports/helpers.py

```
@@ -1,18 +1,13 @@
+from fyle_integrations_imports.modules.base import Base
 
 
-def get_base_class_instance(
-        workspace_id: int = 1,
-        source_field: str = 'PROJECT',
-        destination_field: str = 'PROJECT',
-        platform_class_name: str = 'projects',
-        sync_after: str = None,
-        destination_sync_methods: List[str] = ['customers']
-    ):
+def get_base_class_instance(workspace_id: int = 1, source_field: str = 'PROJECT', destination_field: str = 'PROJECT', platform_class_name: str = 'projects', sync_after: str = None, destination_sync_methods: List[str] = ['customers']):
```

Ruff (0.11.9)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)